pkg/core/waitfor/waitfor.go

```diff
@@ -53,7 +53,7 @@ func (Condition) LoadTestFinished(event watch.Event) (bool, error) {
 
 // Resource waits until a kubernetes resources to match a condition
 func Resource(obj watch.Interface, condFunc watchtools.ConditionFunc) (*watch.Event, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), kubernetes.KubeTimeout)
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
Same for `30` - is it fine to have magic const here or maybe move it to configuration?

Reply:
Done in ae26927